### ISSUE TITLE
need some help here. looks to me that minetweaker3 is to blame 

### ISSUE THREAD

**Issue by PSILONCRAFT** [jaredlll08/ModTweaker#481]
#### Issue Description:
Hi im the modpack author of project psiloncrafted that is available from curseforge and twitch
I updated all my modt through twitch launcher before exporting it and uploading it to curseforge.com

#### What happens:
Server thread/ERROR [modtweaker]: Skipping event FMLModIdMappingEvent and marking errored mod modtweaker since required dependency MineTweaker3 has errored
09.05 10:14:27 [Server] Server thread/ERROR [loottweaker]: Skipping event FMLModIdMappingEvent and marking errored mod loottweaker since required dependency MineTweaker3 has errored
09.05 10:14:27 [Server] Server thread/ERROR [initialinventory]: Skipping event FMLModIdMappingEvent and marking errored mod initialinventory since required dependency MineTweaker3 has errored

#### What you expected to happen:

prior to that we was working on that wither boss did not drop a nether star

#### Minetweaker.log file (Please Pastebin or gist your file, posting an unpasted or ungist'd file will automatically close this issue):

____
#### Affected Versions (Do *not* use "latest"):

- Minecraft:1.10.2
- Forge:12.18.3.2281
- Crafttweaker: 1.10.2-6.1.16
- ModTweaker: 2.2.0.12
- MTLib:1.0.2

#### Your most recent log file where the issue was present: 
[fml-server-1.log.zip](https://github.com/jaredlll08/ModTweaker/files/986001/fml-server-1.log.zip)

**Comment by jaredlll08** [comment]
Uh I'm worry, but what version of modtweaker and crafttweaker are you
using?

**Comment by PSILONCRAFT** [comment]
> - Minecraft:1.10.2
> - Forge:12.18.3.2281
> - Crafttweaker: 1.10.2-6.1.16
> - ModTweaker: 2.2.0.12
> - MTLib:1.0.2

**Comment by jaredlll08** [comment]
Well you are using a crafttweaker that doesn't exist... Please relook at
your versions.

**Comment by PSILONCRAFT** [comment]
<img width="860" alt="skjermbilde 2017-05-09 kl 14 16 57" src="https://cloud.githubusercontent.com/assets/26674039/25850345/3608af78-34c2-11e7-8986-ff0cd03b2b76.png">

**Comment by PSILONCRAFT** [comment]
thats the one

**Comment by jaredlll08** [comment]
Please can I have your client log as well

**Comment by PSILONCRAFT** [comment]
sorry i dont have it. this is a server. but we found out why this happened, quote from the staff:
Ok so I think I know the problem with the crashes we were getting from ModTweaker, well Minetweaker/craftingtweaker things...man too many tweakers...anyway. The wither doesn't have a loottable, well it does but it returns a value of empty, which means that it stores nothing of a loot table, the drop is coded into it. I believe the hardcore with changes it but still pulls from its own classes and drops the loot after rather than from the table. The script I wrote won't work because of these empty loot tables and thus was causing the crash to happen since minetweaker couldn't figure it out. So the solution as far as I can see is to add the mod moreloottables, which adds in a empty but usable loottable so that the script for adding in the star and bags will work.
